course05/homework01/joel/string.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <cstring>
+#include <iostream>
```

Comment:
Why is iostream needed here? You should exclusively include what is needed , nothing else